swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing",condition = 0):
```

Comment:
Based on PEP8 styling guide, "Don’t use spaces around the `=` sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter."

```suggestion
    def __init__(self, category="Clothing", condition=0):
```
[source link](https://peps.python.org/pep-0008/#other-recommendations)